ops/docker-compose.yml

```diff
@@ -35,7 +35,9 @@ services:
         DEPLOYER_PRIVATE_KEY: "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
         SEQUENCER_PRIVATE_KEY: "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
         GAS_PRICE_ORACLE_OWNER: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
-        WHITELIST_OWNER: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
+        # setting the whitelist owner to address(0) disables the whitelist
+        WHITELIST_OWNER: "0x0000000000000000000000000000000000000000"
+        #WHITELIST_OWNER: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
```

Comment:
The commented out line could be deleted correct?

Reply:
Correct, will do.